test/test_datasets.py

```diff
@@ -2095,11 +2100,16 @@ def inject_fake_data(self, tmpdir, config):
 
     @datasets_utils.test_all_configs
     def test_flow(self, config):
+        h, w = self.FLOW_H, self.FLOW_W
+        expected_flow = np.arange(3 * h * w).reshape(h, w, 3).transpose(2, 0, 1)
+        expected_flow = np.flip(expected_flow, axis=1)
+        expected_flow = expected_flow[:2, :, :]
```

Comment:
This is unrelated to the fix, just a slightly more robust way to check the .pfm file decoding. I decided to disagree with myself and my previous comment

> We don't check the values because the reshaping and flipping makes it hard to figure out